app/controllers/ControllerBase.php

```diff
@@ -2,8 +2,34 @@
 
 namespace GanttDashboard\App\Controllers;
 
+use Phalcon\Http\Response;
 use Phalcon\Mvc\Controller;
 
 class ControllerBase extends Controller
```

Comment:
isn't there other solution for this?
also when doing base classes, they are abstract...